src/modules/events/actions/log-handlers.js

```diff
@@ -121,9 +122,10 @@ export const handleTradingProceedsClaimedLog = log => (dispatch, getState) => {
   const isStoredTransaction = log.sender === getState().loginAccount.address
   if (isStoredTransaction) {
     dispatch(updateLoggedTransactions(log))
-    dispatch(loadAccountTrades({ marketId: log.marketId }))
+    dispatch(loadAccountTrades({ marketId: log.market }))
+    dispatch(getWinningBalance([log.market]))
   }
-  if (isCurrentMarket(log.marketId)) dispatch(loadBidsAsks(log.marketId))
+  if (isCurrentMarket(log.market)) dispatch(loadBidsAsks(log.market))
```

Comment:
Are we sure its .market and not .marketId. This emitter is augur-node generated. It doesnt come off contracts.

Reply:
actually this looks good. I was thinking this was market state emitter, my mistake